lib/galaxy/authnz/managers.py

```diff
@@ -113,7 +113,7 @@ def _parse_oidc_backends_config(self, config_file):
                     self.oidc_backends_config[idp] = self._parse_idp_config(child)
                     self.oidc_backends_implementation[idp] = 'psa'
                     self.app.config.oidc[idp] = {'icon': self._get_idp_icon(idp)}
-                elif idp == 'custos' or idp == 'cilogon':
+                elif idp == 'custos' or idp == 'cilogon' or idp == 'keycloak':
```

Comment:
Can you add something like `CUSTOS_BACKENDS = {'custos', 'cilogin', 'keycloak'}` and then use this here and on line 177 and line 273